### ISSUE TITLE
No silent image usage possible

### ISSUE THREAD

**Issue by tobiasd** [friendica/friendica#6167]
### Expected behavior

When uploading a picture at `/photos` and select the option to not send out an posting about the upload no such posting should happen.

### Actual behavior

This works when the picture is uploaded.

But when one later posts something (an article or event) containing the image, a seperate posting containing only the image is created and send to the contacts. This posting is not displayed at the local timeline, only remotely.

(see [this forum entry](https://friendica.kommune4.de/display/4029d2ca135bf0104ad123e314305698) for a description in German)

### Friendica version you encountered the problem

2018.12-dev

**Comment by MrPetovan** [comment]
No activity so far here, pushing to next release.

**Comment by annando** [comment]
Is this still an issue?

**Comment by tobiasd** [comment]
Yes.

**Comment by annando** [comment]
The picture is posted via AP? This is an important details to pin down the problem.

**Comment by tobiasd** [comment]
The network displayed in my test is only "Friendica". I'm not sure for other additional picture postings as I made no notes/or saved links to them.

**Comment by AlfredSK** [comment]
Confirmed

**Comment by tobiasd** [comment]
I just made a test, uploading a picture with "Do not show a status post for this upload" checked and then write a separate posting embedding the image in it. I'm still getting both as contact of the test account, the post only containing the uploaded image, and the actual posting the image was embedded in.

So #6885 did not fix the issue in my test.

**Comment by AlfredSK** [comment]
Hmm. I just did the same test. And it worked for me. No additional post with only the image has been created.

But what didn't work: If I uncheck the "do not show a status..." I didn't get a post either. :-)

**Comment by tobiasd** [comment]
Tested it as well. Forget to actually post something (as I was testing another issue parallel) but then used this mishap to check if the forced checking of new stuff (the update now functionality in `/contact`) and the posted image was fetched as well, even without the posting.

**Comment by tobiasd** [comment]
Seems to be working now for me as well.

**Comment by AlfredSK** [comment]
Ähm. A surprise two days later...

https://libranet.de/display/0b6b25a8-455c-8b78-b074-c80396752313
https://libranet.de/display/0b6b25a8-195c-8b78-369a-f5b263284151

This is one of the test posts I mentioned earlier. There was no additional post for the image upload when I checked it two days ago.
I'll do another test.

**Comment by AlfredSK** [comment]
And like @tobiasd already pointed out the user (author/owner) doesn't see it in the network stream or home/profile stream. It's completely hidden from the author/owner. Only other users (local and remote) see the additional image post on their network stream.

**Comment by MrPetovan** [comment]
Both these posts seem to date back (2019-03-15 10:02:02 UTC) from before the last PR was even submitted (2019-03-15 20:34:00 UTC), which would explain why it still didn't work.

**Comment by tobiasd** [comment]
> A surprise two days later...

I wonder why it took two days. The worker should check the contacts once a day... or did this interval changed?

**Comment by annando** [comment]
The workers check the contact _at least_ once a day. But since this is done with a low priority (40) this can take even longer on busy systems.

**Comment by AlfredSK** [comment]
@tobiasd @annando  Yes. That's possible. My node has more than 30000 jobs with low priorities at the moment.
@MrPetovan Right. But I was surprised nonetheless. :-) I have uploaded and posted a new image. I'll see what happens...

**Comment by tobiasd** [comment]
You could "force check for updates" for the test contact @AlfredSK maybe then the low priority job will be done quicker then.

**Comment by AlfredSK** [comment]
Ok. I tested again and forced the contract update a couple of times in the last 4 hours. No additional image post until now.
